### ISSUE TITLE
"Add Expression" workflow appears to make invalid references?

### ISSUE THREAD

**Issue by jamlung-ri** [OpenConceptLab/ocl_issues#1684]
The "Add Reference(s)" -> "Add Expression" workflow seems to have a bug when using the non-expanded version of the form. I have been unable to successfully add concepts to my collection when inputting what looks like a valid expression into that field. It always appears to end up unresolved for some reason. [Link to collection](https://app.openconceptlab.org/#/users/jamlung/collections/my-labs/references) if helpful.

![image](https://github.com/OpenConceptLab/ocl_issues/assets/55999682/60b9416f-bc47-4979-b897-3da6609a8325)
![image](https://github.com/OpenConceptLab/ocl_issues/assets/55999682/9de55b19-66f9-4076-8f97-00f9bb5339f6)

See the first minute or so of this Loom to see the bug happening: https://www.loom.com/share/8c4e3233f8184716b67e77f172dfc4d4

FYI this is a low priority bug in my opinion since very few people use this workflow, as I understand it. :)

**Comment by snyaggarwal** [comment]
Yep this feels like broken. The request structure its making is not understood by parser. Not sure where this went wrong, I remember this to be working.

Rather than changing the parser or TB request structure, may be we should wait for v3 and see how the UI shapes up?

**Comment by jamlung-ri** [comment]
> Rather than changing the parser or TB request structure, may be we should wait for v3 and see how the UI shapes up?

This feels right to me! Let's save this for v3.

**Comment by paynejd** [comment]
Closing as part of OCL issues cleanup — web2 is deprecated in favor of TBv3/web3. If still relevant, reopen with updated context.
